registration-processor/core-processor/registration-processor-bio-dedupe-stage/src/test/java/io/mosip/registration/processor/biodedupe/stage/BioDedupeProcessorTest.java

```diff
@@ -454,7 +454,7 @@ public void testBioDeDupUpdatePacketHandlerProcessingFailure() throws ApisResour
 		Mockito.when(abisHandlerUtil.getUniqueRegIds(any(),any(),any())).thenReturn(matchedRidList);
 
 		MessageDTO messageDto = bioDedupeProcessor.process(dto, stageName);
-		assertFalse(messageDto.getIsValid());
+		assertTrue(messageDto.getIsValid());
```

Comment:
Why is this changed to asset true ?

Reply:
in this case, request is going to manual verification stage. so according to my latest changes isValid should be true...